buildSrc/src/main/groovy/io.deephaven.java-classpath-conventions.gradle

```diff
@@ -32,12 +32,12 @@ configurations {
     fishNet.extendsFrom fishIo
     fishNumerics.extendsFrom fishBase
     fishUtil.extendsFrom fishConfig
-    fishStats.extendsFrom fishUtil
+    fishNet.extendsFrom fishUtil
```

Comment:
This seems incorrect and/or duplicative (since you have `fishNet.extendsFrom` twice). I think you can delete this line and have `dhUtil` depend on `fishIo`. I think some usages of `:Net` in gradle build files might be overkill.

Not for the PR, but we should be deleting some of this code. Net and a lot of what's in IO should be replaced.

Reply:
Opened #3962 for clean up task. Your gradle suggestions appear to work and are on their way.